HT-3.py

```diff
@@ -0,0 +1,61 @@
+people = [
```

Comment:
This module better remove from current pull request and organize as separate pull request